volatility3/framework/plugins/linux/mountinfo.py

```diff
@@ -1,7 +1,6 @@
 # This file is Copyright 2021 Volatility Foundation and licensed under the Volatility Software License 1.0
 # which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
 #
-# Author: Gustavo Moreira
```

Comment:
Thanks for this, we do request that authors not include their author information to avoid people feeling as though the files can't be modified, or that some see plugins as being owned by specific people.  However, he haven't heavily enforced this, and haven't really developed an AUTHORS file to contain a list of all the contributors.  As such, I'd like to check with @gcmoreira to make sure that's he's happy to have this removed before we commit it.  If not, we'll need to speed up writing an AUTHORS file and pull the information from github commits before we can make changes like this.  The rest of it looks fine to commit...

Reply:
All right, We considered remove it, but I hope one day I have another way to thank him for his wonderful work.

Reply:
Sure, please remove it, no problem ;)

Reply:
Awesome, thanks @gcmoreira !  5:)

Reply:
Thank you, @gcmoreira :)